src/parcsr_ls/par_mgr_setup.c

```diff
@@ -1377,33 +1395,35 @@ hypre_MGRSetupFrelaxVcycleData( void *mgr_vdata,
           /* We need to mask out C-points from outer CF-marker for C/F relaxation at solve phase --DOK*/
           for (i = 0; i < local_size; i++)
           {
-            if (CF_marker_array[lev][i] == 1)
+            if (hypre_IntArrayData(CF_marker_array[lev])[i] == 1)
             {
               CF_marker_local[i] = 0;
             }
           }
-          CF_marker_array_local[lev_local] = CF_marker_local;
+          CF_marker_array_local[lev_local] = hypre_IntArrayCreate(local_size);
```

Comment:
@waynemitchell why is a new IntArray created here again? Is this different from what was created in line 1370?

Reply:
Ah, yes, this is redundant. Thanks for the catch.